contribute-to-timescaledb/contribute-to-docs.md

```diff
@@ -0,0 +1,266 @@
+    docs. They must be fixed before your contribution can be accepted.
+*   Warnings. Less serious than errors. Adopt these recommendations unless you
+    have a strong reason not to.
+*   Suggestions. The least serious level. Consider and adopt these
+    recommendations, unless your content is a special case or exception.
+
+## Structure
+
+### Information when it's needed, where it's needed
+Readers don't read documentation from cover to cover, like a book. They skim and
+skip. Don't assume that they know something because it's explained earlier in
+the document. Use cross-references to guide them to further information when
+they need it.  
+
+### Documenting processes and procedures
+Documentation often describes action steps needed to complete a task. A
+**procedure** is a set of steps needed to achieve a goal. A **process** is a set
+of procedures needed to achieve a larger goal.
+
+In Timescale documentation, procedures have special markup:
+*   They are wrapped in `<procedure>` tags. For more information, see the
+    [markup conventions][markup-procedures].
+*   They start with a heading in gerund form. That means the first word is a
+    verb ending in "-ing."
+*   They contain a numbered list of steps. 
+
+To describe a process, create several child procedures. Then list all the child
+procedures in one place for quick reference.
+
+## Word use
+
+## A [](#A)
+
+*Adverbs*
+: Do not use.
+
+&#128077; Install TimescaleDB.
+
+&#10060; Simply install TimescaleDB.
+
+*And/Or*
+: Do not use. You can usually pick one. If you're not sure, pick "and."
+
+&#10060; I like apples and/or oranges.
+
+&#128077; I like apples and oranges.
+
+## B [](#B)
+
+## C [](#C)
+
+*Contractions*
+: Absolutely fine to use, but try not to overdo it.
+
+## D [](#D)
+
+## E [](#E)
+
+## F [](#F)
+
+*File system*
+: Two words.
+
+## G [](#G)
+
+## H [](#H)
+
+## I [](#I)
+
+## J [](#J)
+
+## K [](#K)
+
+## L [](#L)
+
+*Latin abbreviations*
+: Do not use.
+
+&#128077; For example
+
+&#10060; eg
+
+## M [](#M)
+
+## N [](#N)
+
+## O [](#O)
+
+## P [](#P)
+
+*Please*
+: Do not use. Just state the required action.
+
+## Q [](#Q)
+
+## R [](#R)
+
+## S [](#S)
+
+## T [](#T)
+
+*Timescale*
+: The name of the company. Do not use camel case.
+
+&#128077; Timescale is hosting a virtual event.
+
+&#10060; I have installed Timescale to manage my time-series data.
+
+&#10060; TimeScale
+
+*TimescaleDB*
+: The name of the product. Capitalize the initial letter and the "DB" at the end.
+
+&#128077; "I have installed TimescaleDB to manage my time-series data."
+
+&#10060; "TimescaleDB is hosting a virtual event."
+
+&#10060; "TimeScaleDB"
+
+## U [](#U)
+
+*Utilize*
+: Do not use. Use "use" instead.
+
+## V [](#V)
+
+*Via*
+: Avoid if possible. There is usually a more accurate English word, like
+"through," "with," or "using."
+
+## W [](#W)
+
+*We*
+: Do not use. Use second person ("you") instead.
```

Comment:
I think we can safely say to use this judiciously, and only when we're referring to Timescale (the company). As in, "We strongly recommend you upgraded to the newest version of TimescaleDB", etc. I think it's better to try adn avoid it, generally, but there are definitely places where I would prefer to use "we" over "Timescale" to suit our company voice better.